4. Javascript/4.2. Array and For Loop/1_array_elements.js

```diff
@@ -0,0 +1,6 @@
+let numbers = [5, 9, 3, 19, 70, 8, 100, 2, 35, 27];
+
+// -------------------
+// Nesse primeiro exercício, percorra o array imprimindo todos os valores nele contidos com a função console.log();
+
+console.log(numbers);
```

Comment:
Olá Silvio, a execução do exercício não está correta. Você precisa utilizar o for para percorrer o array. Estou a disposição caso precise de ajuda.

Reply:
Obrigado, Aline! Foi falta de atenção da minha parte! Já vou corrigir!

Reply:
```suggestion
for (let i = 0; i < numbers.length; i += 1) {
  console.log(numbers[i]);
}

```